README.md

```diff
@@ -61,7 +63,7 @@ cfn test --enforce-timeout 60 -- -k contract_delete_update # combine args
 
 To validate the schema, use the `validate` command.
 
-This command is automatically run whenever one attempts to submit a resource or module. Any module fragments will be automatically validated via [`cfn-lint`](https://github.com/aws-cloudformation/cfn-python-lint/), however any warnings or errors detected by [`cfn-lint`](https://github.com/aws-cloudformation/cfn-python-lint/) will not cause this step to fail.
+This command is automatically run whenever one attempts to submit a resource or module. Any module fragments will be automatically validated via [`cfn-lint`](https://github.com/aws-cloudformation/cfn-python-lint/), however any warnings detected will not cause this step to fail.
```

Comment:
nit: cfn-lint warnings will not cause `cfn-validate` to fail. However, other validation errors would still fail `cfn-validate`, right?
The clarification earlier was a clearer message to users.

Reply:
Tweaked the wording in the latest commit.